cmd/tailscale/cli/up.go

```diff
@@ -716,6 +717,7 @@ func init() {
 	addPrefFlagMapping("nickname", "ProfileName")
 	addPrefFlagMapping("update-check", "AutoUpdate")
 	addPrefFlagMapping("auto-update", "AutoUpdate")
+	addPrefFlagMapping("posture-checks", "PostureChecks")
```

Comment:
this is plural "checks" but line 718 is "update-check" singular. Kinda weird being different?

Reply:
hmm, I dont have a strong opinion, but we will be doing multiple checks, so plural makes sense to me. I do not know what its the best choice here with consistency vs language?

Reply:
We could also call this PostureChecking (--posture-checking=true) and not have to make a decision on singular vs plural :)

Reply:
Done